lib/v8.js

```diff
@@ -80,3 +87,110 @@ exports.getHeapSpaceStatistics = function() {
 
   return heapSpaceStatistics;
 };
+
+/* V8 serialization API */
```

Comment:
Never looked at it that way.  I suppose it's fine, lib/ is a mixture of both; `//` is more prevalent but that's probably also because of the copyright header.